src/controllers/eventstreamController.js

```diff
@@ -176,6 +176,7 @@ export async function getEventstream(req, res) {
     let collectionURI = baseURICollection + 'dataset/' + institution + '/' + md5(institution + adlibdatabase);
     let fragmentContent = {
       "@context": {
+        "ldes": "https://ldes.qa.stad.gent/ldes/",

```

Comment:
should be "ldes": "https://w3id.org/ldes#"

Reply:
```suggestion
        "ldes": "https://w3id.org/ldes#",

```